static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
The only reason was that if there's a one-to-one mapping between a piece of markup and its styling, and that styling the markup's raison d'etre, then separating the two across two different files is just diffusing your implementation.

Follow your discretion.